src/infra/docs/bors.md

```diff
@@ -21,10 +21,7 @@ failed are re-approved again on their own.
 
 There are multiple steps needed to add a repository to our Bors instance:
 
-1. The [@bors] GitHub account needs to be granted write access to the
-   repository.
```

Comment:
Why was this removed? Bors still needs write access.

Reply:
I think I didn't remember needing to do this, but I'm probably misremembering